tests/config/test_argparse_config.py

```diff
@@ -43,7 +43,10 @@ def test_logger_commandline() -> None:
     def test_logger_rcfile() -> None:
         """Check that we parse the rcfile for the logging checker correctly."""
         with pytest.raises(SystemExit) as ex:
-            Run([LOGGING_TEST, f"--rcfile={LOGGING_TEST.replace('.py', '.rc')}"])
+            # replace only the last .py in the string with .rc
+            # we do so by inverting the string and replace the first occurrence (of the inverted tokens!)
+            _rcfile = LOGGING_TEST[::-1].replace("yp.", "cr.", 1)[::-1]
```

Comment:
I was searching for an elegant way to do that and saw : https://stackoverflow.com/a/2556156/2519059 😄 You've been there before me 😉 ?

Reply:
yep!